pytorch_lightning/callbacks/stochastic_weight_avg.py

```diff
@@ -187,16 +189,18 @@ def on_train_epoch_start(self, trainer: 'pl.Trainer', pl_module: 'pl.LightningMo
                 anneal_strategy=self._annealing_strategy,
                 last_epoch=trainer.max_epochs if self._annealing_strategy == "cos" else -1
             )
-            _scheduler_config = _get_default_scheduler_config()
-            assert _scheduler_config["interval"] == "epoch" and _scheduler_config["frequency"] == 1
-            _scheduler_config["scheduler"] = self._swa_scheduler
+            default_scheduler_cfg = _get_default_scheduler_config()
+            assert default_scheduler_cfg["interval"] == "epoch" and default_scheduler_cfg["frequency"] == 1
```

Comment:
n00b question: why is this assert statement here? `_get_default_scheduler_config` just returns a `dict` right? Also, if it can sometimes not return a dict, why give error sometimes and warning others?

Reply:
it will return a dict. the assert is there as a sanity check in case somebody modifies it. But it could be removed

Reply:
Makes sense :smiley: